gen/calc.py

```diff
@@ -622,13 +622,11 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
             'rexray': {
                 'loglevel': 'info',
                 'modules': {
-                    'default-admin': {
```

Comment:
Good question! @tuxity was there a specific reason for no longer specifying port?